src/renderers/shared/fiber/ReactFiberCommitWork.js

```diff
@@ -316,7 +316,8 @@ module.exports = function<T, P, I, TI, C>(config : HostConfig<T, P, I, TI, C>) {
         const textInstance : TI = finishedWork.stateNode;
         const newText : string = finishedWork.memoizedProps;
         const oldText : string = current.memoizedProps;
-        commitTextUpdate(textInstance, oldText, newText);
+        const parentInstance : ?I = current.return && current.return.stateNode;
```

Comment:
This is not guaranteed to be a host node. It can be any other type of work. Therefore, this assignment isn't type safe. You'd need to recurse upwards quite a bit to find the parent but we wouldn't want to do that for any text update since this is only needed in an edge case.